mojo/stdlib/stdlib/hashlib/_fnv1a.mojo

```diff
@@ -14,6 +14,7 @@
 """Implements the [Fnv1a 64 bit variant](https://en.wikipedia.org/wiki/Fowler–Noll–Vo_hash_function) algorithm as a Hasher type."""
 
 from .hasher import Hasher
+from memory import Span
```

Comment:
```suggestion
```
You can drop the import as `Span` is part of the `prelude` which means it is automatically imported.